.github/workflows/build.yml

```
@@ -0,0 +1,144 @@
+    runs-on: ubuntu-latest
+    needs: prepare  # 确保这个作业在 prepare 作业完成后运行
+    strategy:
+      matrix:
+        goos: [linux, darwin, freebsd]
+        goarch: [amd64, arm64]
+    env:
+      OUTPUT_BINARY: ghproxy
+      GO_VERSION: 1.24
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)